source/stylesheets/base/_layout.scss

```diff
@@ -51,3 +51,12 @@ body {
     justify-content: space-around;
   }
 }
+
+@media only screen
+  and (max-device-width: 640px),
+  only screen and (max-device-width: 667px),
+  only screen and (max-width: 480px) {
+  .container {
+    max-width: 100%;
+  }
+}
```

Comment:
This screenshot from https://neat.bourbon.io/examples/ might help you think about it.

![screen shot 2017-11-01 at 12 13 47 pm](https://user-images.githubusercontent.com/5566826/32284612-2a0d4e7e-befe-11e7-9f78-e7ffe70dcf0f.png)